src/express/routes/api.ts

```diff
@@ -0,0 +1,53 @@
+import {CodeReviewRelation, PrismaClient, User} from '@prisma/client';
+import express from 'express';
+import {logError} from '../../utils/log';
+
+const api = express.Router();
+
+api.get('/reviews/:status?', (req, res, next) => {
+    const prisma = new PrismaClient();
+
+    const status = req.params.status;
+    const where: {[index: string]: string} = {}
```

Comment:
huh, is this a way to type an object with any number of properties?
that's clever, but would it be more useful in this use case to do
```
const where: { status?: string } = {};
```

Reply:
Yes, this is an indexed dynamic object with an unknown number of properties.